src/VirtoCommerce.SimpleExportImportModule.Data/Validation/ImportProductPricesExistanceValidator.cs

```diff
@@ -0,0 +1,41 @@
+using System.Linq;
```

Comment:
Please rename file itself to ImportProductPricesExistenceValidator.cs

Reply:
Fixed